classes/data-stores/ActionScheduler_DBLogger.php

```diff
@@ -143,4 +148,31 @@ public function bulk_log_cancel_actions( $action_ids ) {
 
 		$wpdb->query( $sql_query );
 	}
+
+	/**
+	 * Update the logs table schema to allow the datetime fields to be NULL.
+	 * The NOT NULL causes a conflict with some versions of MySQL configured with sql_mode=NO_ZERO_DATE.
+	 *
+	 * @param string $table Name of table being updated.
+	 * @param string $db_version The existing schema version of the table.
+	 */
+	public function update_logs_schema_4_0( $table, $db_version ) {
```

Comment:
I think we could move this method to a new class or file, just to avoid having this kind of method inside this class, since it could get bigger in the future.
Also this method doesn't seems to require anything from this class and could be hooked `action_scheduler_before_schema_update` from anywhere.

Reply:
@claudiosanches what are your thoughts on moving it to the existing `schema/ActionScheduler_LoggerSchema` class (and similarly so for the corresponding `DBStore` method, except in that case we'd use `schema/ActionScheduler_StoreSchema`)?

Reply:
☝🏽 Updated per my last note, let me know your thoughts!